### ISSUE TITLE
Queries fail parsing even though extension is installed

### ISSUE THREAD

**Issue by guomo** [brianc/node-postgres#1646]
I've installed the _intarray_ extension via

`CREATE EXTENSION intarray;`

Queries against the DB parse fine with tools like DataGrip or  psql and produce the desired results. In this case the single bar operator merges two int arrays, de-duping the results and returning a single array. But a node-posgres query like this

```query = `INSERT INTO entities.${entity_str} (source_pk, raw_name, machine_aliases, is_most_wanted) VALUES($1,$2,$3,$4) 
                    ON CONFLICT (raw_name)
                    DO UPDATE SET altsources_pk = ARRAY[EXCLUDED.source_pk] | movie_title.altsources_pk RETURNING id`;```

Fails parsing with the following error:

`error: operator does not exist: integer[] | integer[]`

This extension is part of the standard PG distro so I was expecting the feature to work. Is there any way I can tell node-postgres to recognize the extension? Or some alternate way to submit the query where it skips a parsing step in your code but sends the whole thing to the server without attempting to validate?

**Comment by charmander** [comment]
> Or some alternate way to submit the query where it skips a parsing step in your code but sends the whole thing to the server without attempting to validate?

That’s already what pg does. I’m not sure how it could be affecting your query.

Are you sure you’re running the query against the right database? The operator works for me when attempting `SELECT ARRAY[1] | $1::int[]`, for example.

**Comment by guomo** [comment]
@charmander Yes I'm using the correct database! I believe the issue might be because of the upsert. Note how my query is doing the DO UPDATE SET. Perhaps the pg code is trying to parse that and doesn't recognize the single | operator.
